WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -40,6 +40,28 @@ class WordPressAppDelegate: UIResponder, UIApplicationDelegate {
         return ContextManager.shared.mainContext
     }
 
+    private lazy var abTesting: ExPlat? = {
+        struct WordPressExPlatConfiguration: ExPlatConfiguration {
+            let platform = "calypso"
+
+            var oAuthToken: String? {
+                let accountService = AccountService(managedObjectContext: ContextManager.shared.mainContext)
+                let defaultAccount = accountService.defaultWordPressComAccount()
+                return defaultAccount?.authToken
+            }
+
+            var userAgent: String? {
+                return WPUserAgent.wordPress()
+            }
+
+            var anonId: String? {
+                return UserDefaults.standard.string(forKey: "TracksAnonymousUserID")
+            }
+        }
+
+        return ExPlat(configuration: WordPressExPlatConfiguration())
```

Comment:
My idea is to comment this line and only enable it when there are experiments running.

This avoids unnecessary requests.

Reply:
I wonder if we can take a small step away from commenting this line on/off but maybe instead use an enum or struct to A) provide a key for the experiment B) maintain a list of running experiments. 

So a base enum could look like this:
```
enum ExPlatExperiment: String, CaseIterable {
    /// This is a default experiment here to support a base enum
    case unknown = "FallbackExperiment"
}

extension ExPlatExperiment {
    static var hasRunningExperiments: Bool {
        return ExPlatExperiment.allCases.count > 1
    }
}
```

And when we're running an experiment it would be:
```
enum ExPlatExperiment: String, CaseIterable {
    /// This is a default experiment here to support a base enum
    case unknown = "FallbackExperiment"

    /// Running experiments
    case myTest = "my-awesome-test"
}

extension ExPlatExperiment {
    static var hasRunningExperiments: Bool {
        return ExPlatExperiment.allCases.count > 1
    }
}
```

In these cases `unknown` is just there because Swift doesn't like an empty enum. We could use `hasRunningExperiments` to turn the network call off if desired.

----
If you like that idea but don't like the `unknown` item we can remove `unknown` by going to a struct like:

```
struct ExPlatExperiment {
    /// Running experiments
    static let myTest = "my-awesome-test"
    static let runnningExperiment = [myTest]
}

extension ExPlatExperiment {
    static var hasRunningExperiments: Bool {
        return ExPlatExperiment.runnningExperiment.count > 0
    }
}
```